dev/fattest.simplicity/src/componenttest/topology/utils/tck/TCKRunner.java

```diff
@@ -241,7 +241,7 @@ public TCKRunner withLogging(Map<String, Level> logs) {
         }
 
         // Configure a simple formatter
-        loggingProperties.put("java.util.logging.SimpleFormatter.format", "[%1$tm/%1$te/%1$tY %1$tT:%1$tM %1$tZ] %2$s %4$.1s %5$s %6$s %n");
+        loggingProperties.put("java.util.logging.SimpleFormatter.format", "[%1$tm/%1$td/%1$tY %1$tH:%1$tM:%1$tS %1$tZ] %4$.1s %2$s %n %5$4s %6$4s %n");
```

Comment:
```suggestion
        loggingProperties.put("java.util.logging.SimpleFormatter.format", "[%1$tm/%1$td/%1$tY %1$tH:%1$tM:%1$tS:%1$tL %1$tZ] %4$.1s %2$s %n %5$4s %6$4s %n");
```

Do you want milliseconds here as well?

I would also generally avoid the newline in the middle since it makes it harder to search, but I don't know what sort of messages you're logging with this pattern, so it may still make sense here.

Reply:
Added in the milliseconds.

Had the new line in there so I could read the logs without having to scroll horizontally for 3 years to get past the package/class/method names but do understand it makes the logs harder to parse.  I removed that